packages/fxa-content-server/app/images/send_tab_android.svg

```diff
@@ -1 +0,0 @@
-<svg height="113" width="360" xmlns="http://www.w3.org/2000/svg"><style>@keyframes bob{0%,to{transform:translate(0,0) scale(-1)}50%{transform:translate(0,5px) scale(-1)}}</style><linearGradient id="a" x1="12.402%" x2="87.598%" y1="100%" y2="0%"><stop offset="0" stop-color="#ff7139"/><stop offset=".22" stop-color="#ff624a"/><stop offset=".66" stop-color="#ff3c75"/><stop offset="1" stop-color="#ff298a"/></linearGradient><g fill="none" fill-rule="evenodd"><path d="M338.826.275a2.82 2.82 0 00-.44-.12l-.4-.14a6.54 6.54 0 00-.88 0 6.26 6.26 0 00-.88 0l-.42.14a3.6 3.6 0 00-.44.12 3.42 3.42 0 00-.46.26l-.32.16a3.74 3.74 0 00-.68.58l-13.58 13.56a4.525 4.525 0 006.4 6.4l5.86-5.86v21.84a4.52 4.52 0 009.04 0v-21.88l5.84 5.86a4.525 4.525 0 106.4-6.4l-13.56-13.56a3.86 3.86 0 00-.7-.58l-.3-.16z" fill="url(#a)" fill-rule="nonzero" style="animation:bob .5s infinite;transform-origin:94% 20%" transform="scale(-1)"/><path d="M0 57h360v56H0z" fill="#f0f0f4" fill-rule="nonzero"/><path d="M336 81c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zM291.222 95h-9.444A3.778 3.778 0 01278 91.222v-9.444A3.778 3.778 0 01281.778 78h9.444A3.778 3.778 0 01295 81.778v9.444A3.778 3.778 0 01291.222 95zm-9.365-15A1.857 1.857 0 00280 81.857v9.286c0 1.026.831 1.857 1.857 1.857h9.286A1.857 1.857 0 00293 91.143v-9.286A1.857 1.857 0 00291.143 80zm.143-3a2 2 0 012-2h7.987a6 6 0 016 6v.016l-.024 7.987a2 2 0 01-2.006 1.994l.03-9.986V81a4 4 0 00-4-4z" fill="#0c0c0d" fill-opacity=".6"/><rect fill="#e0e0e6" fill-rule="nonzero" height="40" rx="8" width="248" x="8" y="65"/></g></svg>
```

Comment:
I'm not sure if we ever ended up using these (I don't see them when searching, anyway), but we probably also want to leave the `send_tab` SVGs. I animated these 5 or 6 months for us for a "let's teach the user how to use send tab" project. Maybe check with @davismtl. 🤔

Reply:
Skipped to be safe. They're not harming anything.